pkg/noderesourcetopology/plugin.go

```diff
@@ -118,7 +119,7 @@ func New(args runtime.Object, handle framework.Handle) (framework.Plugin, error)
 		return nil, err
 	}
 
-	nrtCache, err := initNodeTopologyInformer(tcfg, handle)
+	nrtCache, err := initNodeTopologyInformer(tcfg, handle, topologyHandle)
```

Comment:
I may prefer to use once.Do() to ensure initNodeTopologyInformer() is only called once, and then share the variable across different plugin instances.

Reply:
yes, the idea is good, updated. PTAL, thanks.